### PR TITLE
chore: updated ubuntu 22.04.1

### PR DESCRIPTION
**Pull request by burnsjared0415**
# 

## Summary of Pull Request
Updated ubuntu version from 22.04 to 22.04.1

## Type of Pull Request

- [ ] This is a bugfix. `type/bug`
- [ ] This is an enhancement or feature. `type/feature` or `type/enhancement`
- [ ] This is a documentation update. `type/docs`
- [ ] This is a refactoring update. `type/refactor`
- [x] This is a chore. `type/chore`
- [ ] This is something else.
      Please describe:

## Related to Existing Issues

Issue Number: N/A

## Test and Documentation Coverage

- [x] Tests have been completed.
- [x] Documentation has been added or updated.

## Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### CODE REVIEW

File: builds/linux/almalinux/8/linux-almalinux.pkrvars.hcl.example [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 // Guest Operating System Metadata
 vm_guest_os_name     = "almalinux"
-vm_guest_os_version  = "8.9"
+vm_guest_os_version  = "8.10"
```
Comment (tenthirtyam) [inline builds/linux/almalinux/8/linux-almalinux.pkrvars.hcl.example:11]: Did you mean to change this here?

File: builds/linux/oracle/8/linux-oracle.pkrvars.hcl.example [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 // Guest Operating System Metadata
 vm_guest_os_name     = "oracle"
-vm_guest_os_version  = "8.9"
+vm_guest_os_version  = "8.10"
```
Comment (tenthirtyam) [inline builds/linux/oracle/8/linux-oracle.pkrvars.hcl.example:12]: Did you mean to change this here?

**Review by tenthirtyam — COMMENTED** [review]
Left two comments.

File: builds/linux/almalinux/8/linux-almalinux.pkrvars.hcl.example (hunk 1 above)
Comment (burnsjared0415) [reply builds/linux/almalinux/8/linux-almalinux.pkrvars.hcl.example:11]: i did, i noticed it was missed by me in the past, i can add it to the changelog

File: builds/linux/oracle/8/linux-oracle.pkrvars.hcl.example (hunk 2 above)
Comment (burnsjared0415) [reply builds/linux/oracle/8/linux-oracle.pkrvars.hcl.example:12]: i did, i noticed it was missed by me in the past, i can add it to the changelog

**Review by tenthirtyam — APPROVED** [review]

**Comment by tenthirtyam** [comment]
@burnsjared0415 - can you check to ensure your commits are signed? Having an issue merging this one.

**Pull request merged**